wormhole-connect/src/utils/inProgressTxCache.ts

```diff
@@ -9,6 +9,89 @@ const LOCAL_STORAGE_MAX = 3;
 const JSONReplacer = (_, value: any) =>
   typeof value === 'bigint' ? value.toString() : value;
 
+// Checks the existance of the given prop in a parent object
+const validateChildProp = (
+  parent: object,
+  propName: string,
+  propType: string,
+) => {
+  return (
+    propName in parent &&
+    parent[propName] !== null && // Line below is an explicit check for undefined values as well
+    typeof parent[propName] === propType
```

Comment:
not positive but I think you only need this line in here? the first two are redundant with it. or am I wrong?